armada/handlers/tiller.py

```diff
@@ -153,8 +153,25 @@ def _pre_update_actions(self, actions, namespace):
                 if "job" in action_type:
                     LOG.info("Deleting %s in namespace: %s", name, namespace)
                     self.k8s.delete_job_action(name, namespace)
-                    continue
-                LOG.error("Unable to execute name: %s type: %s ", name, type)
+                elif "pod" in action_type:
+                    # create label_selector string
+                    label_selector = 'release_name={}'.format(release_name)
```

Comment:
for this pod deletion logic is okay, but i would move it to its own func. i know i would like to use it in #105